app/views/audit.scala.html

```diff
@@ -472,7 +472,11 @@ <h2 id="pop-up-message-title" class="bold">Title</h2>
                         <p>@Messages("audit.left.ui.jump.title")</p>
                     </div>
                     <div id="modal-skip-box-content">
-                        <input type="button" class="overlay-option" id="modal-skip-explore" value="@Messages("audit.left.ui.jump.button.explore")">
+                        @if(user) {
```

Comment:
I think you should be able to turn this into a single if statement, with something like `@if(user.isEmpty || user.get.role.getOrElse("") != "Turker")`.

Reply:
Ok, for sure. However, there are some other instances similar to this one where the if statement is broken up (e.g. lines 516 and 572), so should I fix those as well?

Reply:
Sure, though in those places, I think it should be `@if(user.isDefined && user.get.role.getOrElse("") == "Turker")`. If you change those, just test to make sure everything shows up correctly for anon users, registered users, and turkers.

Reply:
I ended up just removing the `@if(user)` if statement completely, because if 'user' is empty, wouldn't the `@if(user.get.role.getOrElse("") != "Turker")` if statement be true anyways?
Also, I decided to just stick to editing code specific to this issue and not change all of the other similar if statements because there are a large amount of them in audit.scala.html and in the other scala.html files as well.

Reply:
Ooo no if `user` is empty, then `user.get` will throw an exception and the page will just crash. So you only want to run `user.get` if you know that `user` is defined.